applications/ChimeraApplication/custom_strategies/strategies/fs_strategy_for_chimera.h

```diff
@@ -0,0 +1,836 @@
+        const array_1d<double,3> Zero(3,0.0);
+        for ( auto itElem = rModelPart.Elements().ptr_begin(); itElem != rModelPart.Elements().ptr_end(); ++itElem )
+        {
+            if(!(*itElem)->Is(ACTIVE))
+            {
+                for(auto& node : (*itElem)->GetGeometry()){
+                    node.FastGetSolutionStepValue(VELOCITY)  = Zero;
+                    node.FastGetSolutionStepValue(PRESSURE)  = 0.0;
+                }
+            }
+        }
+    }
+
+    void InitializeStrategy(SolverSettingsType& rSolverConfig,
+            bool PredictorCorrector)
+    {
+       KRATOS_TRY;
+
+        BaseType::mTimeOrder = rSolverConfig.GetTimeOrder();
+
+        // Check that input parameters are reasonable and sufficient.
+        BaseType::Check();
+
+        //ModelPart& rModelPart = BaseType::GetModelPart();
+
+        BaseType::mDomainSize = rSolverConfig.GetDomainSize();
+
+        BaseType::mPredictorCorrector = PredictorCorrector;
+
+        BaseType::mUseSlipConditions = rSolverConfig.UseSlipConditions();
+
+        BaseType::mReformDofSet = rSolverConfig.GetReformDofSet();
+
+        BaseType::SetEchoLevel(rSolverConfig.GetEchoLevel());
+
+        // Initialize strategies for each step
+        bool HaveVelStrategy = rSolverConfig.FindStrategy(SolverSettingsType::Velocity,BaseType::mpMomentumStrategy);
+
+        if (HaveVelStrategy)
+        {
+            rSolverConfig.FindTolerance(SolverSettingsType::Velocity,BaseType::mVelocityTolerance);
+            rSolverConfig.FindMaxIter(SolverSettingsType::Velocity,BaseType::mMaxVelocityIter);
+            KRATOS_INFO("FSStrategyForChimera ")<<"Velcoity strategy successfully set !"<<std::endl;
+        }
+        else
+        {
+            KRATOS_THROW_ERROR(std::runtime_error,"FS_Strategy error: No Velocity strategy defined in FractionalStepSettings","");
+        }
+
+        bool HavePressStrategy = rSolverConfig.FindStrategy(SolverSettingsType::Pressure,BaseType::mpPressureStrategy);
+
+        if (HavePressStrategy)
+        {
+            rSolverConfig.FindTolerance(SolverSettingsType::Pressure,BaseType::mPressureTolerance);
+            rSolverConfig.FindMaxIter(SolverSettingsType::Pressure,BaseType::mMaxPressureIter);
+
+            KRATOS_INFO("FSStrategyForChimera ")<<"Pressure strategy successfully set !"<<std::endl;
+        }
+        else
+        {
+            KRATOS_THROW_ERROR(std::runtime_error,"FS_Strategy error: No Pressure strategy defined in FractionalStepSettings","");
+        }
+
+        // Check input parameters
+        BaseType::Check();
+
+        KRATOS_CATCH("");
+    }
+
+
+    ///@}
+    ///@name Private  Access
+    ///@{
+
+
+    ///@}
+    ///@name Private Inquiry
+    ///@{
+
+
+    ///@}
+    ///@name Un accessible methods
+    ///@{
+
+    /// Assignment operator.
+    FSStrategyForChimera& operator=(FSStrategyForChimera const& rOther){}
+
+    /// Copy constructor.
+    FSStrategyForChimera(FSStrategyForChimera const& rOther){}
```

Comment:
As suggested before, I'd move these to the life cycle section of the file and use the `delete`.

Reply:
done